osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/pom.xml

```diff
@@ -77,11 +78,29 @@
     </plugins>
   </build>
 
+  <!-- This repository was added to be able to draw the snapshot jDLMS version from artifactory -->
+  <repositories>
+    <repository>
+      <snapshots/>
+      <id>osgp-snapshots</id>
+      <name>osgp-snapshots</name>
+      <url>http://artifactory.shs.osgp.cloud/artifactory/osgp-snapshots</url>
```

Comment:
It seems that I'm unable to access this repository from my local system and therefore cannot download the depedency with version 1.7.1-SNAPSHOT

Reply:
I can access the repo with my browser though and the version is there.